features/step_definitions/work_steps.rb

```diff
@@ -6,6 +6,77 @@
 DEFAULT_CONTENT = "That could be an amusing crossover."
 DEFAULT_CATEGORY = "Other"
 
+### Setting up a work
+# These steps get used a lot by many other steps and tests to create works in the archive to test with
+
+When /^I fill in the basic work information for "([^\"]*)"$/ do |title|
+  step %{I fill in basic work tags}
+  check(DEFAULT_WARNING)
+  fill_in("Work Title", with: title)
+  fill_in("content", with: DEFAULT_CONTENT)
+end
+
+# Here we set up a draft and can then post it as a draft, preview and post, post without preview, 
+# or fill in additional information on the work form. 
+# Example: I set up the draft "Foo"
+# Example: I set up the draft "Foo" with fandom "Captain America" in the collection "MCU Stories" as a gift to "Bob"
+# 
+# This is a complex regexp because it attempts to be flexible and match a lot of options (including using a/the, in/to etc)
```

Comment:
I know regular expressions are not obvious to everyone, but I think it's reasonable to expect coders reading this to look them up rather than documenting features of the language in our comments. (not blocking, obviously, just a remark inspired by the current trend of "self-documenting" code)